indico/modules/events/abstracts/templates/management/overview.html

```diff
@@ -1,118 +1,155 @@
 {% extends 'events/abstracts/management/_base.html' %}
 
 {% block content %}
-    {% if not cfa.has_started %}
-        {% include 'events/abstracts/management/cfa_actions/not_started.html' %}
-    {% elif cfa.has_started and not cfa.has_ended %}
-        {% include 'events/abstracts/management/cfa_actions/open.html' %}
-    {% elif cfa.has_ended %}
-        {% include 'events/abstracts/management/cfa_actions/ended.html' %}
-    {% endif %}
-
-    <div class="action-box">
-        <div class="section">
-            <div class="icon icon-upload"></div>
-            <div class="text">
-                <div class="label">
-                    {% trans %}Submission{% endtrans %}
-                </div>
-                {% trans %}Configure the submission dates, the abstract fields, etc.{% endtrans %}
-            </div>
-            <div class="toolbar">
-                <div class="group">
-                    <a href="#" class="i-button icon-mail"
-                       data-href="{{ url_for('.email_tpl_list', event) }}"
-                       data-title="{% trans %}Configure e-mail notifications{% endtrans %}"
-                       data-ajax-dialog>
-                        {% trans %}Notifications{% endtrans %}
-                    </a>
+    {% if not event.has_feature('abstracts') %}
+        <div class="action-box highlight">
+            <div class="section">
+                <div class="icon icon-file-check"></div>
+                <div class="text">
+                    <div class="label">
+                        {% trans %}Call for Abstracts disabled{% endtrans %}
+                    </div>
+                    {% trans %}"Call for Abstracts" provides an integrated abstract management and reviewing module that will allow you to set up a full workflow.{% endtrans %}
```

Comment:
line is way too long. split in multiple lines (don't forget whitespace trimming in the trans tags)